hack/ci-set-org-vars.sh

```diff
@@ -130,9 +139,13 @@ setVars() {
 setVar() {
     NAME=$1
     VALUE=$2
-    echo -n "Setting $NAME in $GIT_ORG: "
-    "${SCM}SetVar"
-    sleep .5 # rate limiting to prevent issues
+    echo -n "Setting $NAME: "
+    if [ -n "${DRY_RUN:-}" ]; then
+        echo "'$VALUE'"
+    else
+        "${SCM}SetVar"
+        sleep .5 # rate limiting to prevent issues
```

Comment:
So no need to wait when dry run is true? Still don't see why we need to wait at all.

Reply:
No need to wait on dry-run. The goal of the sleep is to throttle the calls, I believe I put that there after I got errors during testing. It's not new though.

Reply:
sounds good